raiden/transfer/node.py

```diff
@@ -951,7 +951,7 @@ def handle_state_change(
         iteration = handle_init_target(chain_state, state_change, storage, chain_state.our_address)
     elif type(state_change) == ActionInitTargetLight:
         assert isinstance(state_change, ActionInitTargetLight), MYPY_ANNOTATION
-        iteration = handle_init_target(chain_state, state_change, storage, state_change.transfer.initiator)
+        iteration = handle_init_target(chain_state, state_change, storage, state_change.transfer.target)
```

Comment:
This is at least confusing, but i suspect that it will break light payments reception:

- `ActionInitTargetLight ` is the state that is generated on a light client payment reception on a single hop payment. That is a feature that is currently working and tested. Making this change could affect that.
- `handle_init_target `receives an initiator address, so at least is the parameter name confusing.
- `handle_init_target` invokes `subdispatch_targettask` by `initiator address`, so its also confusing.

Reply:
As we spoke yesterday for this case. 
In the ActionInitTarget, the handle_init_target function gets chain_state.our_address that corresponds as it says the node itself address. This address is used to get a channel_state, from here we can see that in Light case, we get the initiator address.
The function that is using this addresses is get_channelstate_by_canonical_identifier_and_address, and if you send an address from a partner it will first try to get the LC address with that address and a channel identifier, and then take a look at channelidentifiers_to_channels with the lc_address obtained. We take the decision of letting this change so the two cases ActionInitTargetLight and ActionInitTarget are going to behave the same, we can send the initiator as it was, but this will cause confuse situations like this.

Reply:
Agree.

We still have to modify the mentioned methods to receive a more representative parameter name for the address instead of `initiator`, but i agree with the change. 

Also i want to make mention that since we know the `channel_id ` and a member of the channel (therefore we can use `get_channelstate_by_canonical_identifier_and_address` ), using `initiator `or `target `has the same effect.